detection_rules/etc/non-ecs-schema.json

```diff
@@ -125,5 +125,9 @@
   },
   "logs-ti_*": {
     "labels.is_ioc_transform_source": "keyword"
+  },
+  "logs-auditd_manager.auditd-*": {
+    "auditd.data.terminal": "keyword",
+    "auditd.data.addr": "keyword"
```

Comment:
We don't need to include these. The reason your rule may have been failing prior to this addition is because `event.dataset` is not set in the query, thus the rule validation could not identify the right package and integration to validate fields on. Adding them to `non-ecs-schema.json` bypasses this validation entirely.